src/ensembl/src/content/app/species/state/sidebar/speciesSidebarSelectors.ts

```diff
@@ -17,8 +17,14 @@
 import { getActiveGenomeId } from 'src/content/app/species/state/general/speciesGeneralSelectors';
 import { RootState } from 'src/store';
 
-export const isSidebarOpen = (state: RootState) =>
-  state.speciesPage.sidebar.isOpen;
+export const isSidebarOpen = (state: RootState) => {
+  const activeGenomeId = getActiveGenomeId(state);
+
+  if (!activeGenomeId) {
+    return false;
+  }
+  return !!state.speciesPage.sidebar.species[activeGenomeId]?.isSidebarOpen;
```

Comment:
It's strange to see the boolean (`isSidebarOpen`) converted to a boolean with the `!!`.
Don't you mean return `state.speciesPage.sidebar.species[activeGenomeId]?.isSidebarOpen ?? false`?

Reply:
That's done